bin/templates/cordova/lib/android_sdk.js

```diff
@@ -62,7 +62,8 @@ module.exports.version_string_to_api_level = {
     '5.1': 22,
     '6.0': 23,
     '7.0': 24,
-    '7.1.1': 25
+    '7.1.1': 25,
+    '8.0': 26
```

Comment:
API 27 is out as well, right - which one does that correspond to? 8.1?

Reply:
Correct!  That should be added as well, it seems.  See also, my #430.